pom.xml

```diff
@@ -532,6 +547,11 @@
                 <artifactId>flyway-core</artifactId>
                 <version>6.1.3</version>
             </dependency>
+            <dependency>
+                <groupId>org.mongodb</groupId>
+                <artifactId>mongodb-driver</artifactId>
+                <version>3.11.1</version>
```

Comment:
any specific reason to use 3.x instead of 4.x?

Reply:
This latest mongodb-driver version in the maven central repository is 3.12.2 (March 2020). There seems to be no version 4.x in the maven central repository.